regexes/oss.yml

```diff
@@ -303,23 +303,75 @@
 ##########
 # iOS
 ##########
-- regex: 'CFNetwork/889'
+- regex: 'CFNetwork/1206.*Darwin/20.1.0(?!.*x86_64)'
```

Comment:
cfnetwork version is no needed, change to
'CFNetwork/.+ Darwin/20.1.0(?!.*x86_64)'